packages/hint-manifest-scoped-urls/tests/tests.ts

```diff
@@ -0,0 +1,84 @@
+import { test } from '@hint/utils';
```

Comment:
I think we should also test the local connector but I'm not sure how to check that things are correct because paths can be anything locally 🤔
@antross do you have any suggestion?